Slash/Utility/Comments/Comments.pm

```diff
@@ -1345,44 +935,21 @@ sub displayThread {
 			$class = 'hidden';
 		} elsif ($comment->{points} < $threshold) {
 			if ($user->{is_anon} || ($user->{uid} != $comment->{uid})) {
-				if ($discussion2) {
-					$class = 'hidden';
-					$hidden++;
-				} else {
-					$hidden++;
-					next;
-				}
+				$hidden++;
+				next;
 			}
 		}
 
 		my $highlight = ($comment->{points} >= $highlightthresh && $class ne 'hidden') ? 1 : 0;
 		$class = 'full' if $highlight;
-		if ($discussion2 && $user->{state}{d2_defaultclass}{$cid}) {
-			$class = $user->{state}{d2_defaultclass}{$cid};
-		}
 		$comment->{class} = $class;
 
 		$user->{state}{comments}{totals}{$class}++ unless $comment->{dummy};
 
 		my $finish_list = 0;
 
-		if ($full || $highlight || $discussion2) {
-			if ($discussion2 && $class eq 'oneline' && $comment->{subject_orig} eq 'no') {
-				$comment->{subject} = 'Re:';
-			}
-
+		if ($full || $highlight) {
```

Comment:
Everything should be sent as full.  we should fix this and get rid of extra checks.